Tests fail: 'No Reader has been initialized for this Clip' and 'File could not be opened'

Hi,
I'm  unable to make tests pass on my system. I just ran `cmake . && make && tests/openshot-test` on an  up-to-date git clone and I see the following:

```
----------------------------
     RUNNING ALL TESTS
----------------------------
/tmp/libopenshot/tests/Clip_Tests.cpp:50:1: error: Failure in Clip_Constructor: Unhandled exception: No Reader has been initialized for this Clip.  Call Reader(*reader) before calling this method.
/tmp/libopenshot/tests/Clip_Tests.cpp:205:1: error: Failure in Clip_Effects: Unhandled exception: No Reader has been initialized for this Clip.  Call Reader(*reader) before calling this method.
/tmp/libopenshot/tests/ImageWriter_Tests.cpp:36:1: error: Failure in ImageWriter_Test_Gif: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/FFmpegReader_Tests.cpp:41:1: error: Failure in FFmpegReader_GetFrame_Before_Opening: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/FFmpegReader_Tests.cpp:52:1: error: Failure in FFmpegReader_Check_Audio_File: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/FFmpegReader_Tests.cpp:82:1: error: Failure in FFmpegReader_Check_Video_File: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/FFmpegReader_Tests.cpp:120:1: error: Failure in FFmpegReader_Seek: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/FFmpegReader_Tests.cpp:177:1: error: Failure in FFmpegReader_Multiple_Open_and_Close: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/FFmpegWriter_Tests.cpp:35:1: error: Failure in FFmpegWriter_Test_Webm: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/FrameMapper_Tests.cpp:180:1: error: Failure in FrameMapper_resample_audio_48000_to_41000: Unhandled exception: File could not be opened.
/tmp/libopenshot/tests/Timeline_Tests.cpp:88:1: error: Failure in Timeline_Check_Two_Track_Video: Unhandled exception: No Reader has been initialized for this Clip.  Call Reader(*reader) before calling this method.
/tmp/libopenshot/tests/Timeline_Tests.cpp:210:1: error: Failure in Timeline_Clip_Order: Unhandled exception: No Reader has been initialized for this Clip.  Call Reader(*reader) before calling this method.
/tmp/libopenshot/tests/Timeline_Tests.cpp:438:1: error: Failure in Timeline_Effect_Blur: Unhandled exception: No Reader has been initialized for this Clip.  Call Reader(*reader) before calling this method.
FAILURE: 13 out of 74 tests failed (13 failures).
Test time: 0.33 seconds.
----------------------------
```

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -24,15 +24,18 @@
 # along with OpenShot Library. If not, see <http://www.gnu.org/licenses/>.
 ################################################################################
 
+SET(TEST_MEDIA_PATH "${openshot_SOURCE_DIR}/src/examples/")
 
 ################ WINDOWS ##################
 # Set some compiler options for Windows
 # required for libopenshot-audio headers
 IF (WIN32)
-	STRING(REPLACE "/" "\\\\" TEST_MEDIA_PATH "${openshot_SOURCE_DIR}/src/examples/")
-	add_definitions( -DIGNORE_JUCE_HYPOT=1 -DTEST_MEDIA_PATH="${TEST_MEDIA_PATH}" )
+	STRING(REPLACE "/" "\\\\" TEST_MEDIA_PATH TEST_MEDIA_PATH)
+	add_definitions( -DIGNORE_JUCE_HYPOT=1 )
 	SET(CMAKE_CXX_FLAGS " ${CMAKE_CXX_FLAGS} -include cmath")
 ENDIF(WIN32)
+
+add_definitions( -DTEST_MEDIA_PATH="${TEST_MEDIA_PATH}" )
 
 ################### UNITTEST++ #####################
 # Find UnitTest++ libraries (used for unit testing)

